.github/workflows/release.yaml

```
@@ -0,0 +1,127 @@
+
+    # Step to upload the tar.gz package as an artifact
+    - name: Upload build artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: OpenConverter_Linux_x86
+        path: OpenConverter_Linux_x86.tar.gz
+
+
+    - name: Get GitHub Release information
+      id: release_info
+      run: echo "::set-output name=release_tag::$(echo ${GITHUB_REF#refs/tags/})"
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2116:style:1:38: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'

(shellcheck)